storage-provider/server/src/indexer/local_index_directory/rdb.rs

```diff
@@ -601,6 +601,7 @@ impl Service for RocksDBLid {
     fn add_index(
         &self,
         piece_cid: Cid,
+        roots: Vec<Cid>,
```

Comment:
This diverges from the original reference, could you add the usefulness of the `roots` to the docstring?